bindings/python/test/geometry/d2/objects/test_polygon.py

```diff
@@ -267,8 +258,8 @@ def test_getters (self):
 
         # Inner Ring Vertices
         point_5: Point = Point(-0.5, 0.5)
-        point_6: Point = Point(0.5, 0.5)
-        point_7: Point = Point(0.5, -0.5)
+        point_6: Point = Point(0.5, -0.5)
+        point_7: Point = Point(0.5, 0.5)
```

Comment:
Why the change? (do we break the test otherwise?)

Reply:
Yes we do, this is linked to the ring ordering which is expected in boost to have valid polygons and that we don't enforce. We can find a way to keep the flexibility on our side while converting when needed and going through boost. I have a ticket to check/tackle that one separately